components/avatar/style/index.tsx

```diff
@@ -2,4 +2,5 @@ import '../../style/index.less';
 import './index.less';
 
 // style dependencies
+// deps-lint-skip: grid
```

Comment:
What does this line for?

Reply:
Due to the fact I'm using `useBreakpoint`, which comes from `grid`, the linter fails since it requires the style for grid to be imported within `avatar` folder as well, however we're not using anything that depends on style from grid so I guess we're fine with just skipping it